SeleniumBasicsTest/appSource/app_tecklist.py

```diff
@@ -0,0 +1,72 @@
+from selenium.webdriver.support.select import Select
+
+from libCommon.selenium_helper import SeleniumHelper
+import time
+from sourcelocators.techlocators import Techlistlocators
+
+class APPTechList:
+
+    def __init__(self):
+        self.url = "https://www.techlistic.com/p/selenium-practice-form.html"
+        self.driver = SeleniumHelper()
+        self.driver.implecit_waits(10)
+
+    def launch(self):

```

Comment:
separate modules define to launch application and form filling
ex: def launch_app(self): once its launch then def form_filling(self): perform actions what ever you want here

Reply:
i will change as per the error and i will PR
thanks very much